test/contracts/Wallet.cpp

```diff
@@ -458,6 +458,11 @@ class WalletTestFramework: public SolidityExecutionFramework
 		sendMessage(*s_compiledWallet + args, true, _value);
 		BOOST_REQUIRE(!m_output.empty());
 	}
+
+	Address emptyAccount()
+	{
+		return Address(m_rpc.accountCreate());
```

Comment:
You know that this takes about  a second to run?

Reply:
There's the alternative solution in commit one. Handling the starting balance.

Reply:
@chriseth so which one should we have:
1) take starting balance and compare to that
2) send off all the balance to somewhere (ie 0)
3) create empty account

The problem with 2) is that `sendEther` doesn't currently enforces it was successful, though we can add another check to see if the balance is 0 after.

Reply:
4. just use a different address than 5?